test/UnitTest/Microsoft.AspNet.OData.Test/packages.config

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Castle.Core" version="4.2.0" targetFramework="net45" />
+  <package id="EntityFramework" version="6.4.4" targetFramework="net452" />
```

Comment:
>EntityFramework" [](start = 15, length = 16)

why do you add EF in the unit test?

you can move your test to E2E if it's related to Entity Frame